src/figuresClasses.ts

```diff
@@ -1,11 +1,106 @@
-export interface Figure {}
+/* eslint-disable @typescript-eslint/lines-between-class-members */
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+  private sideA: number;
+  private sideB: number;
+  private sideC: number;
+
+  constructor(
+    public color: Color,
+    sideA: number,
+    sideB: number,
+    sideC: number,
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
+      throw new Error('your error message');
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message.
![image](https://github.com/mate-academy/ts_figures_classes/assets/63243136/96ae4247-eda7-4436-96dc-5e05120098bc)

Fix it in all places